.azdo/pipelines/azure-dev.yml

```diff
@@ -7,6 +7,10 @@ pool:
 container: mcr.microsoft.com/azure-dev-cli-apps:latest
 
 steps:
+  - pwsh: |
+      azd config set auth.useAzCliAuth "true"
```

Comment:
dachte dises flag braucht es seit dem fix nicht mehr?

Reply:
Good catch. That is basically a copy&paste form the original template located here: 
<https://github.com/Azure-Samples/todo-nodejs-mongo/blob/main/.azdo/pipelines/azure-dev.yml>

Maybe a azdo limitation?

Reply:
And yes it is a limitation of the Azure DevOps pipeline as referenced here <https://github.com/Azure/azure-dev/issues/1126>